tests/bootstrap.php

```diff
@@ -7,7 +7,9 @@
 require __DIR__.'/dummy/Queries/MoviesFrom2014Query.php';
 require __DIR__.'/dummy/Queries/MoviesFromXYearQuery.php';
 require __DIR__.'/dummy/Queries/MovieWithIDXQuery.php';
+require __DIR__.'/dummy/Queries/PaginatedMoviesFrom2014Query.php';
 require __DIR__.'/dummy/Queries/CountMoviesFrom2014Query.php';
+require __DIR__.'/dummy/Queries/SortedQuery.php';
 require __DIR__.'/dummy/Fragments/Filters/IDFilter.php';
 
 // Test host (should be inside the vagrant box).
```

Comment:
you can also use `autoload-dev` in composer.json to make this a PSR-4 namespaced folder. In a different PR ofc.